app/javascript/src/Navigation/components/VerticalNav.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+import React from 'react'
+import { Nav, NavExpandable, NavItem, NavList, NavGroup } from 'Navigation/components/PF4NavProxy'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+
+type Item = {
+  id: string,
+  title: string,
+  path: string,
+  target: ?string
+}
+
+type Section = {
```

Comment:
Not super useful since we have only 2, but an interface for the types in this case could be good

Reply:
Both `Item` and `Section` have almost the same props

Reply:
I'll change to a union type.